src/main/java/manufacturing/dao/jdbc/ManufacturerDaoJdbc.java

```diff
@@ -0,0 +1,119 @@
+package manufacturing.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import manufacturing.dao.ManufacturerDao;
+import manufacturing.exception.DataProcessingException;
+import manufacturing.injections.Dao;
+import manufacturing.model.Manufacturer;
+import manufacturing.util.ConnectionController;
+
+@Dao
+public class ManufacturerDaoJdbc implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        try (Connection connection = ConnectionController.getConnection()) {
+            String query = "INSERT INTO manufacturers (name, country) VALUES (?, ?)";
+            PreparedStatement insert =
+                    connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
```

Comment:
```suggestion
        String query = "INSERT INTO manufacturers (name, country) VALUES (?, ?)";
        try (Connection connection = ConnectionController.getConnection();
                    PreparedStatement insertStatement =
                            connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
```
and then we can remove `insert.close()`